runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/BpmnModeler/BpmnModeler.types.ts

```diff
@@ -291,5 +291,6 @@ export enum ModelerEvent {
     PROPERTIESPANEL_PROVIDERSCHANGED = 'propertiesPanel.providersChanged',
     CANVAS_RESIZED = 'canvas.resized',
     IMPORT_PARSE_COMPLETE = 'import.parse.complete',
+    IMPORT_DONE = 'import.done',
```

Comment:
we fill attended and trigger details on process import - when does it happen? maybe its better to have output at the same place

Reply:
I think, this event of BPMN Modeler can be misleading, it appears every time we import or save process, but also every time the modeler viewport is loaded